### PR TITLE
picture fail

### PR DESCRIPTION
**Pull request by stacyburris**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [[x]]Picture of whiteboard

insertion

### CODE REVIEW

File: javascript/linkedList/README.md [hunk 1]
```diff
@@ -30,3 +30,4 @@
 + Sara Strasner and Tina Myers
 
 + ![LL Insert](/assets/insertion.png)
```
Comment (adrianhuebner) [inline javascript/linkedList/README.md:32]: This line should be:

`LL Insert`

Comment (stacyburris) [reply javascript/linkedList/README.md:32]: Thank you! All fixed
